paper.tex

```diff
@@ -949,6 +962,35 @@ \subsubsection{Editing a Tour}
 	\label{fig:editour}
 \end{figure}
 
+\subsubsection{Editour Frontend Implementation}
+\label{sec:editourFrontendImplementation}
+
+The frontend was implemented as a web app, using web technologies such as HTML,
+CSS and JavaScript, with JavaScript being the vast majority. ECMAScript is the
+standard that modern JavaScript follows.
+
+Type-checking provided by Visual Studio Code and JSDoc made the growing code
+complexity more manageable~\footnote{One helpful specification of the ECMAScript
+standard in the way of programmer sanity is ``strict mode.'' By placing the
+string \texttt{"use strict"} at the top of a JavaScript source file, errors will
+be thrown in many cases where ordinary JavaScript would not complain. The
+simplest example of this is the line \texttt{a = 1;} will not throw an error
+ordinarily. In strict mode, you must at first explicitly declare the variable
+with the \texttt{let}, \texttt{var} or \texttt{const} keyword before assigning
+it, similar to Java or C++. We included the string at the beginning of every
+source file except for class declarations, which are in strict mode by
+default~\autocite[386]{ecmainternational2019}.} . Excluding Node.js modules, the
+only external library we used for Editour was Leaflet, an open-source JavaScript
+library for interactive maps~\autocite{leafletjs}. We did not use any framework
```

Comment:
"for **creating** interactive maps"

Reply:
replaced it with a quote from the website